webpack.config.ts

```diff
@@ -41,8 +75,8 @@ const config: Configuration = {
             ".mjs": [".mjs", ".mts"]
         }
     },
-    devtool: "source-map",
+    //devtool: "none",
```

Comment:
Is there a reason for this change?

Reply:
ahh, good catch, that was a left-over from a test (because disabling this, speeds up webpack as it does not have to produce source maps).

will revert back to "source-map" for now (in the future I will propose to change it to https://github.com/TriliumNext/Notes/pull/1225/commits/f18866833afbfd43172cbacfdbb8da5dc3410371 – but that is part of a different scope of work)